src/common/gui/MSEGEditor.cpp

```diff
@@ -1800,9 +1811,8 @@ int32_t MSEGControlRegion::controlModifierClicked(CControl* pControl, CButtonSta
          {
             auto val = op.second;
             auto men = addcb(op.first, [val, pControl, this]() {
-               pControl->setValue(val);
-               canvas->invalid();
-               invalid();
+               pControl->setValue(Parameter::intScaledToFloat(val, 100, 1));
```

Comment:
Uhhh ... this is wrong. What about the controls which aren't 100->1 CNumberFields where the value is 0.5?

Reply:
Fixing based on our Discord chat! :)